server/src/helpers/construct-tx.ts

```diff
@@ -76,6 +77,15 @@ export function parseTransaction(
   const tx = signed
     ? decode(transaction, { registry, metadataRpc })
     : decode(JSON.parse(hexToString(transaction)), { registry, metadataRpc });
+
+  tx.metadataRpc = "0x...truncated...";
```

Comment:
The `metadataRpc` field is pretty big and we don't currently care what it is right now, so just cutting it down. It enforces being 0x-prefixed, which is why I left that there.